src/main/java/org/isf/medicalinventory/model/MedicalInventory.java

```diff
@@ -34,9 +37,6 @@
 import jakarta.persistence.Version;
 import jakarta.validation.constraints.NotNull;
 
-import org.isf.utils.db.Auditable;
-import org.springframework.data.jpa.domain.support.AuditingEntityListener;
-
```

Comment:
The import order was correct before you change; please the ide_settings directory for the correct ordering.